platforms/core-runtime/gradle-cli-main/build.gradle.kts

```diff
@@ -15,14 +15,27 @@
  */
 
 plugins {
-    id("gradlebuild.distribution.api-java")
+    id("gradlebuild.distribution.implementation-java")
+    id("gradlebuild.launchable-jar")
+    id("gradlebuild.start-scripts")
 }
 
-description = "Java 6-compatible entry point of the `gradle` command. See :launcher project for the rest."
+description = "Java 6-compatible entry point of the `gradle` command. Boostraps the Gradle client implementation in :launcher."
```

Comment:
Nothing to fix here but I find it interesting only the `gradle` command has a java 6 compatible entry-point, and the wrapper itself does not